react-components/src/analysis/messages.js

```diff
@@ -49,6 +50,11 @@ var intlData = {
         viewAll: "View All analyses",
         analysisInfoDlgTitle: "Analysis Info",
         copyAnalysisId: "Copy Analysis ID",
+        followLogs: "Follow Logs",
+        jobLogsUnavailableMessage:
+            "Logs will be available under your output folder after status of" +
+            " this analysis changes to Completed or Failed.",
```

Comment:
Maybe we could update this to something like `The logs will be available within the analysis output folder after the status of this analysis changes to Completed or Failed.`